src/css/video-js.less

```diff
@@ -964,33 +964,92 @@ body.vjs-full-window {
 /* Text Track Styles */
 /* Overall track holder for both captions and subtitles */
 .video-js .vjs-text-track-display {
-  text-align: center;
   position: absolute;
-  bottom: 4em;
-  /* Leave padding on left and right *///
-  left: 1em;
-  right: 1em;
+  top: 0;
+  left: 0;
+  bottom: 3em;
+  right: 0;
+  pointer-events: none;
+}
+
+/* Increase font-size when fullscreen */
```

Comment:
Hm... I think this should be removed, though, it's actually not doing anything right now because vttjs overrides the font-size manually. I think it got added when I merged some stuff from aster.